pyalgostrategypool/options_bear_call_ladder_v2.py

```diff
@@ -0,0 +1,130 @@
+from constants import *
+from strategy.core.strategy_options_base_v2 import StrategyOptionsBaseV2
+
+
+class StrategyOptionsBearCallLadder(StrategyOptionsBaseV2):
+    name = 'Options Bear Call Ladder Template v2'
+
+    def __init__(self, *args, **kwargs):
+        """
+        Accept and sanitize all your parameters here.
+        Setup the variables you will need here.
+        If you are running the strategy for multiple days, then this method will be called only once at the start of the strategy.
+        """
+
+        super().__init__(*args, **kwargs)
+
+        # Parameters (currently set with default values, can be overridden from the Parameters Configuration Pane)
```

Comment:
`# Parameters (currently set with default values, can be overridden from the Parameters Configuration Pane)`

keep this comment for all options strategies

Reply:
also keep the in-line comments below that explain the different values

Reply:
done